example/src/components/SingleProductReview/index.tsx

```diff
@@ -38,6 +38,18 @@ export const SingleProductReview = (props: SingleProductReviewParams) => {
       justifyContent: 'center',
       marginVertical: 10,
     },
+    logoWrapper: {
```

Comment:
Create a styled component instead to declare this style. The function of this definition is only to modify styles ready for example: a button style